.kiro/specs/1.0-procedural-characters.md

```
@@ -0,0 +1,134 @@
+# Spec: 1.0 Procedural Character System
+
+## Overview
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)